keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -506,6 +503,36 @@ def _parse_backbone(backbone, include_rescaling, backbone_weights):
     return backbone
 
 
+def _parse_box_loss(loss):
+    if not isinstance(loss, str):
+        # support arbitrary callables
+        return loss
+
+    # case insensitive comparison
+    if loss.lower() == "smoothl1":
+        return keras_cv.losses.SmoothL1Loss(l1_cutoff=1.0, reduction="none")
+
+    raise ValueError(
+        "Expected `box_loss` to be either a Keras Loss, "
+        f"callable, or the string 'SmoothL1'.  Got loss={loss}."
+    )
+
+
+def _parse_classification_loss(loss):
+    if not isinstance(loss, str):
+        # support arbitrary callables
+        return loss
+
+    # case insensitive comparison
+    if loss.lower() == "focal":
+        return keras_cv.losses.FocalLoss(from_logits=True, reduction="none")
```

Comment:
Do we check `from_logits` if the user pass a callable?

Reply:
No we don't, should we just accept loss objects?

Reply:
I think loss objects was added to Keras *after* loss callables, with intention to be the default, so yeah only accepting objects is fine with me